tests/conftest.py

```diff
@@ -259,6 +259,12 @@ def outliers_tsds():
     return tsds
 
 
+@pytest.fixture()
+def outliers_tsds_without_missing(outliers_tsds):
+    tsds = TSDataset(df=outliers_tsds[..., "target"].dropna(), freq="1d", df_exog=outliers_tsds.df_exog.dropna())
```

Comment:
Why `freq="1d"` instead of `freq="D"`?

Reply:
These notations are identical. It just left after copying from the fixture above.